raw_data/preprocessed/africa/tanzania_english.txt

```diff
@@ -0,0 +1,769 @@
+
+Final disposal of solid waste 
+(119) The local government authorities shall, in choosing the best method of solid waste disposal for their geographical areas of jurisdiction, consider the following matters: [a] climatic conditions; [b] economic ability; [c] interest of the community; [d] environmental, hygienic and social benefits; and [e] availability of tipping sites. 
+
+[b] Management of Litter
+
+Interpretation with regard to management of litter 
+(120) For the purposes of sections 121 and 122, unless the context otherwise requires "public land" means land entrusted to the President as trustee and includes general land or privately occupied land; "deposit", in relation to litter; include: [a] cast, place, throw or drop litter; [b] allow litter to be cast, thrown, dropped, or to escape, from any conveyance; "litter" includes any refuse, rubbish, animal remains, glass, metal, plastics, garbage, debris, dirt, filth, urine, rubble, ballasts, stones, earth, sewage, or waste matter, or any other thing of a like nature; "private land" means every place other than a public place; "public place": [a] includes the following: [i] every road, street, private street, footpath, access way, storm water drain, service lane, court, mall, thoroughfare, wharf and airport to which the public generally has access, whether with or without payment of any fee, and includes any national park or reserve; [ii] any park, garden or other place of public recreation to which the public has access, whether with or without payment of any fee; [iii] any beach or foreshore, or the bank of any river or stream, or the margin of any lake, to which the public traditionally has access, whether with or without payment of fee; [iv] any waters to which the public traditionally has access, whether with or without payment of any fee, for bathing or other recreational purposes; [v] every wharf, pier or jetty to which the public has access; [vi] any airport; [vii] any general land vested in or controlled by the President being land that is not occupied pursuant to any lease, licence, or other authority by any private person; [viii] any other place whether public or private in the open air to which the public has access, whether with or without payment of any fee; but [b] excludes any site designated or approved for the disposal of waste pursuant to this or any other Act, or any receptacle installed in a public place. 
```

Comment:
Please replace `in relation to litter; include` with `in relation to litter, includes`.